### PR TITLE
Refactored upload tab functionality

### PR DESCRIPTION
**Pull request by akhilraghavakalal**
Refactored the upload tab functionality to handle multiple images and made the UI consistent

### CODE REVIEW

File: backend/src/models/product.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 from bson import ObjectId
-from typing import List
+from typing import List,Optional
 
 class ProductModel(BaseModel):
     id: str = Field(default_factory=lambda: str(ObjectId()), alias="_id")
     product_name: str
     description: str
     price: float
+    seller_id:  Optional[str] = None
```
Comment (durgakarituri23) [inline backend/src/models/product.py:10]: Does this show seller specific products?

Comment (akhilraghavakalal) [reply backend/src/models/product.py:10]: Yes Durga, this field is used to handle the task where an seller can see only the products that are uploaded by them, not other seller products

Comment (durgakarituri23) [reply backend/src/models/product.py:10]: hoo kk, looks good merge it now!!!

**Comment by akhilraghavakalal** [comment]
Approved, merging to main

**Pull request merged**